server/neptune/gateway/event/comment_handler.go

```diff
@@ -38,27 +44,30 @@ func NewCommentEventWorkerProxy(
 	allocator feature.Allocator,
 	scheduler scheduler,
 	rootDeployer rootDeployer,
+	templateLoader template.Loader[any],
```

Comment:
instead of any shouldn't it be the type of the data container you're passing in? I guess in this case nil?

Reply:
According to the go compiler, `nil` is not a type which makes sense since this is basically telling the compiler what can be passed in as `templateLoader` and I think configuring it to `template.Loader[any]` allows us to pass in any instantiation for the Loader. Also, the interface that we use for `templateLoader`
```
Load(id template.Key, repo models.Repo, data any) (string, error)
```
uses `any` data type so it makes sense to use `any` for the generic I think.

Reply:
> Also, the interface that we use for `templateLoader` uses `any` data type so it makes sense to use `any` for the generic I think.

You created the interface for this implementation though, it's not like this has other implementations so I don't get this point.  I don't really even understand why we need the interface here tbh, one unnecessary step of indirection.

I'm fine with leaving it as any here but just calling that out.

Reply:
Removed the interface and defined an empty `Input` type to be used by the loader.